forte/data/readers/audio_reader.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2022 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+The reader that reads audio files into Datapacks.
+"""
+import os
+from typing import Any, Iterator
+import soundfile
```

Comment:
so this reader would depend on the `soundfile` dependency, which means it needs to be in our core requirement. So what happens would be if we do `pip install forte` and `from forte.data.readers.misc_reader import xxx`, this would fail (since this is in the `__init__.py`)

We need to think of a better way to place this reader. Any suggestions